crates/bevy_tasks/src/task_pool.rs

```diff
@@ -584,7 +582,7 @@ impl Default for TaskPool {
 
 impl Drop for TaskPool {
     fn drop(&mut self) {
-        self.shutdown_tx.close();
+        self.shutdown.notify(self.threads.len());
```

Comment:
might be worth trying usize::MAX here and seeing if it helps.

Reply:
I already tried this above and it doesn't seem to work.